docs/user/upgrade.rst

```diff
@@ -141,6 +141,12 @@ The signature for ``org.locationtech.geomesa.security.AuthorizationsProvider#con
 interfaces will need to update their method signature. Any classes invoking these methods should not need to updated,
 as the new signature is compatible with the old one.
 
+Accumulo Default Visibilities Removed
+-------------------------------------
+
+The Accumulo data store parameter ``geomesa.security.visibilities`` have been removed. Visibilities should be set
+per-feature, as per :ref:`accumulo_visibilities`.
```

Comment:
Are per-feature visibilities only described in the Accumulo section (as this seems to imply)?

Reply:
Currently we have a section in each data store and no generic section. That might be worth revisiting since it is largely the same across data stores. I opened a ticket: https://geomesa.atlassian.net/browse/GEOMESA-2830